coldfront/core/project/templates/project/project_removal/project_removal_request_list.html

```diff
@@ -74,114 +74,23 @@ <h1>Project Removal Requests</h1>
-                        </td>
-                        <td>
-                            {% with status=removal_request.status.name %}
-                                {% if status == "Pending" %}
-                                    <span class="badge badge-warning">{{ status }}</span>
-                                {% elif status == "Processing" %}
-                                    <span class="badge badge-warning">{{ status }}</span>
-                                {% elif status == "Complete" %}
-                                    <span class="badge badge-success">{{ status }}</span>
-                                {% endif %}
-                            {% endwith %}
-                        </td>
-                        {% if user.is_superuser and request_filter == 'pending' %}
-                            <td class="text-nowrap">
-                                {% if removal_request.status.name == "Pending" %}
-                                    <a href="{% url 'project-removal-request-update-status' removal_request.pk %}" class="btn btn-success mr-1">
-                                        Update
-                                    </a>
-                                {% elif  user.is_superuser and removal_request.status.name == "Processing" %}
-                                    <a href="{% url 'project-removal-request-complete-status' removal_request.pk %}" class="btn btn-success mr-1">
-                                        Update
-                                    </a>
-                                {% endif %}
-                            </td>
-                        {% endif %}
-
-                    </tr>
-                {% endfor %}
-                </tbody>
-            </table>
-            {% if is_paginated %} Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}
-                <ul class="pagination float-right mr-3">
-                    {% if page_obj.has_previous %}
-                        <li class="page-item"><a class="page-link" href="?page={{ page_obj.previous_page_number }}&{{filter_parameters_with_order_by}}">Previous</a></li>
-                    {% else %}
-                        <li class="page-item disabled"><a class="page-link" href="#">Previous</a></li>
-                    {% endif %}
-                    {% if page_obj.has_next %}
-                        <li class="page-item"><a class="page-link" href="?page={{ page_obj.next_page_number }}&{{filter_parameters_with_order_by}}">Next</a></li>
-                    {% else %}
-                        <li class="page-item disabled"><a class="page-link" href="#">Next</a></li>
-                    {% endif %}
-                </ul>
-            {% endif %}
+          {% with queryset=removal_request_list %}
+            {% include 'project/project_removal/project_removal_request_list_table.html' %}
+          {% endwith %}
+          {% if is_paginated %} Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}
```

Comment:
The difference between this block and the refactored template seems to be the filter parameters in the `hrefs`. Would those be hard to refactor?

Reply:
We can use the refactored one as long as we don't pass the page_num variable.